packages/grid/_modules_/grid/components/cell/GridRowCells.tsx

```diff
@@ -72,32 +74,40 @@ export const GridRowCells = React.memo((props: RowCellsProps) => {
 
     const cellParams: GridCellParams = apiRef!.current.getCellParams(id, column.field);
 
-    let cssClassProp = { cssClass: '' };
+    const classNames = [cellClassName];
+
     if (column.cellClassName) {
-      if (!isFunction(column.cellClassName)) {
-        cssClassProp = { cssClass: clsx(column.cellClassName) };
-      } else {
-        cssClassProp = { cssClass: column.cellClassName(cellParams) as string };
-      }
+      classNames.push(
+        clsx(
+          isFunction(column.cellClassName)
+            ? column.cellClassName(cellParams)
```

Comment:
should this one still be a function?

Reply:
I think that the cost to keep it is low and to do a conditional formatting in only one column is easier with it than with `getCellClassName`. I would keep it.